config/application.yml.default

```diff
@@ -156,7 +156,6 @@ in_person_outage_expected_update_date: 'October 31, 2024'
 in_person_outage_emailed_by_date: 'November 1, 2024'
 in_person_send_proofing_notifications_enabled: false
 in_person_stop_expiring_enrollments: false
-include_slo_in_saml_metadata: false
```

Comment:
[thought] i think we can't drop the config value until after the code change is deployed. [50-50 state deployment is affected by the config](https://handbook.login.gov/articles/manage-50-50-state.html#change-a-feature-flag-or-config-value) -- can you check to see if we need to keep this in here for deployment, and then have another PR to clean it up?

Reply:
I'm not 100% confident, but I think it'd be okay, since the old boxes would still have both the config value and code paths using the config?

Reply:
Thanks for catching this.

Reply:
Digging into this further, I agree with @aduth that the each instance would either have the config value and code that uses it, or neither. _In theory_, we should be good. What do you all think?

Reply:
i don't know anything about how this works, so if you both are reasonably confident, i'm happy to trust you.

Reply:
Just hopping on to confirm, dropping a default config value in a PR is safe, because as previously stated, each box has its own in-memory copy of the config.

Removing the value from the S3 in the deployed env before this code is deployed: unsafe (because we may need to roll back, then the re-deployed old code would look for configs that aren't there)